From fda0d25c116e920f3df50d51a2cac5eeb118fe60 Mon Sep 17 00:00:00 2001
From: Orivej Desh <orivej@gmx.fr>
Date: Sat, 25 Feb 2017 12:00:11 +0000
Subject: [PATCH 2/6] Sort dialogs by name

---
 Telegram/SourceFiles/dialogswidget.cpp | 8 ++++----
 1 file changed, 4 insertions(+), 4 deletions(-)

diff --git a/Telegram/SourceFiles/dialogswidget.cpp b/Telegram/SourceFiles/dialogswidget.cpp
index b4a02dd..4b29b5c 100644
--- a/Telegram/SourceFiles/dialogswidget.cpp
+++ b/Telegram/SourceFiles/dialogswidget.cpp
@@ -70,13 +70,13 @@ struct DialogsInner::PeerSearchResult {
 };
 
 DialogsInner::DialogsInner(QWidget *parent, QWidget *main) : SplittedWidget(parent)
-, _dialogs(std_::make_unique<Dialogs::IndexedList>(Dialogs::SortMode::Date))
+, _dialogs(std_::make_unique<Dialogs::IndexedList>(Dialogs::SortMode::Name))
 , _contactsNoDialogs(std_::make_unique<Dialogs::IndexedList>(Dialogs::SortMode::Name))
 , _contacts(std_::make_unique<Dialogs::IndexedList>(Dialogs::SortMode::Name))
 , _addContactLnk(this, lang(lng_add_contact_button))
 , _cancelSearchInPeer(this, st::dialogsCancelSearchInPeer) {
 	if (Global::DialogsModeEnabled()) {
-		_dialogsImportant = std_::make_unique<Dialogs::IndexedList>(Dialogs::SortMode::Date);
+		_dialogsImportant = std_::make_unique<Dialogs::IndexedList>(Dialogs::SortMode::Name);
 		_importantSwitch = std_::make_unique<ImportantSwitch>();
 	}
 	connect(main, SIGNAL(peerNameChanged(PeerData*, const PeerData::Names&, const PeerData::NameFirstChars&)), this, SLOT(onPeerNameChanged(PeerData*, const PeerData::Names&, const PeerData::NameFirstChars&)));
@@ -897,9 +897,9 @@ void DialogsInner::onParentGeometryChanged() {
 }
 
 void DialogsInner::onPeerNameChanged(PeerData *peer, const PeerData::Names &oldNames, const PeerData::NameFirstChars &oldChars) {
-	_dialogs->peerNameChanged(Dialogs::Mode::All, peer, oldNames, oldChars);
+	_dialogs->peerNameChanged(peer, oldNames, oldChars);
 	if (_dialogsImportant) {
-		_dialogsImportant->peerNameChanged(Dialogs::Mode::Important, peer, oldNames, oldChars);
+		_dialogsImportant->peerNameChanged(peer, oldNames, oldChars);
 	}
 	_contactsNoDialogs->peerNameChanged(peer, oldNames, oldChars);
 	_contacts->peerNameChanged(peer, oldNames, oldChars);
-- 
2.11.0

